### DISCUSSION TITLE
Problem when using schedules roasts with several stores

### DISCUSSION THREAD

**Discussion by MKSwing** [artisan-roaster-scope/artisan#1699] [General]
When you have two stores and mix scheduled roasts from different stores, sometimes the **template** indicated during scheduling is not loaded by artisan. I am not sure but I think it's happening when your next scheduled roast is not from the same store.

**Comment by MAKOMO** [comment]
Artisan will not find roast profiles (or templates) that have been moved to a different file location after last open or save. To fix, just load those profiles from their new location to make Artisan aware of their new file locations.
